contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/statsbeat_metrics/statsbeat.py

```diff
@@ -33,9 +35,12 @@
 _DEFAULT_STATS_LONG_EXPORT_INTERVAL = 86400  # 24 hours
 
 _ATTACH_METRIC_NAME = "Attach"
+_REQ_SUC_COUNT_NAME = "Request Success Count"
```

Comment:
what about the rest of the network count? failure, duration avg, throttle, exception and retry?

Reply:
I only added success count so the PR would be easier to read, as outlined in the description. Will be adding the rest in a followup PR.

Reply:
sure.